test/NuGet.Core.Tests/NuGet.Protocol.Core.v3.Tests/HttpSource/HttpRetryHandlerTests.cs

```diff
@@ -132,14 +182,14 @@ public class HttpRetryHandlerTests
             var retryDelay = TimeSpan.FromMilliseconds(50);
```

Comment:
more of an unrelated to this PR, why do we have 50 and 100 millisecond delays for different scenarios? Can we commonize them and put them in one variable